etcdserver/raft.go

```diff
@@ -176,6 +184,11 @@ func (r *raftNode) apply() chan apply {
 }
 
 func (r *raftNode) stop() {
+	r.stopped <- struct{}{}
+	<-r.done
+}
+
+func (r *raftNode) close() {
```

Comment:
can we rename this to shutdown or something else with a comment? this confuses me with `stop`.